game/game.py

```diff
@@ -4,27 +4,14 @@
 
 def main():
     """Runs the whole program."""
-    score = [0, 0]
-    choices = {'1': 'scissors',
-               '2': 'rock',
-               '3': 'paper'}
     while True:
         print(get_info())
-        user_choice = get_user_choice()
-        comp_choice = get_comp_choice(1, 3)
-        print_made_choices(user_choice, str(comp_choice), choices)
-        winner = make_decision(user_choice, comp_choice)
-        score = update_score(score, winner)
-        if winner != 'draw':
-            print("Winner: {0}".format(winner))
-        print('####Score####', '\nUser: {0}\nComputer: {1}'.format(*score))
-        exit_program()
-        print('\n')
-
-
-def print_made_choices(user_choice, comp_choice, choices):
+        print_report()
+
+
+def get_made_choices(user_choice, comp_choice, choices):
     """Printing user's and computer's choices."""
-    print("Your choice was: \"{0}\", computer choice was: \"{1}\"".format(choices[user_choice], choices[comp_choice]))
+    return "Your choice was: \"{0}\", computer choice was: \"{1}\"".format(choices[user_choice], choices[comp_choice])
```

Comment:
we don't want to return message, we want to return values, why do I need that text??
I want this function to return made choices without anything else